src/c/admin-contribution-detail.js

```diff
@@ -35,7 +35,11 @@ window.c.AdminContributionDetail = (function(m, _, c, h){
             outerLabel: 'Recompensa',
             getModel: c.models.rewardDetail,
             updateModel: c.models.contributionDetail,
-            selectedItem: reward
+            selectedItem: reward,
+            validate: (newRewardValue) => {
+              return (args.item.value > newRewardValue) ? true : false;
+            },
+            validateErrorMessage: 'Valor mínimo da recompensa é maior do que o valor da contribuição.'
```

Comment:
I would change the validate interface and return this error message.
When validate function return undefined is because everything went fine.
The function would be something like:
```javascript
validate: (newRewardValue) => {
return (args.item.value > newRewardValue) ? undefined : 'Valor mínimo da recompensa é maior do que o valor da contribuição.';
}
```